src/pages/workspaces/workspace/Analyses.js

```diff
@@ -304,17 +306,19 @@ const Analyses = _.flow(
         reportError('Error creating analysis', error)
       }
     }
-  })
+  }) : () => {}
 
   // Lifecycle
   useOnMount(() => {
     const load = async () => {
```

Comment:
I think you need to wrap these Ajax calls with `withErrorReporting`. Otherwise, if an error occurs with one of them, a rejected promise results and you can end up with integration test failures with no error messages. For example, we just saw this type of failure on a run-notebook test, and in the console there was this 500 error:

page.http POST	200  https://terra-bard-dev.appspot.com/api/event
page.http GET	200  https://leonardo.dsde-dev.broadinstitute.org/api/google/v1/runtimes?saturnAutoCreated=true&googleProject=terra-dev-f6ee1cfe&creator=scarlett.flowerpicker%40test.firecloud.org
page.http POST	500  https://sam.dsde-dev.broadinstitute.org/api/google/user/petServiceAccount/terra-dev-f6ee1cfe/token
page.console ConsoleMessage {
  _type: 'error',
  _text: 'Failed to load resource: the server responded with a status of 500 ()',
  _args: [],
  _stackTraceLocations: [
    {
      url: 'https://sam.dsde-dev.broadinstitute.org/api/google/user/petServiceAccount/terra-dev-f6ee1cfe/token',
      lineNumber: undefined
    }
  ]
}

I think that all of our Ajax calls should either be wrapped with `withErrorIgnoring` (if the error is not important and we don't want to notify the user, for instance in the case of NPS survey or metrics logging not working), or `withErrorReporting`.

Reply:
Ah ok. Based on this logic it would seem both could use it :)

Reply:
Yes!